src/object/obj_verify.c

```diff
@@ -254,6 +256,9 @@ dc_obj_verify_parse_akey(struct dc_obj_verify_args *dova, daos_obj_id_t oid,
 	    !daos_key_match(&cursor->iod.iod_name, &akey)) {
 		daos_iov_free(&cursor->iod.iod_name);
 		daos_iov_copy(&cursor->iod.iod_name, &akey);
+		memset(cursor->iod.iod_recxs, 0,
+		       sizeof(*cursor->iod.iod_recxs));
+		cursor->iod.iod_size = DAOS_SIZE_MAX;
```

Comment:
Why need this? the "cursor->iod.iod_size" is initialized at the beginning of the dc_obj_verify_move_cursor() that will call dc_obj_verify_parse_akey().

Reply:
hmm, you are right, this is not needed. I had thought to resolve that "initializing recx, iod_size" for re-enumerate path,  we probably should not set iod_type in that case. Thanks.